examples/cleanrl_examples/ppo_atari_envpool.py

```diff
@@ -220,25 +212,37 @@ def __init__(self, env, deque_size=100):
     self.num_envs = getattr(env, "num_envs", 1)
     self.episode_returns = None
     self.episode_lengths = None
-    self.is_vector_env = True
+    # get if the env has lives
+    self.has_lives = False
+    env.reset()
+    info = env.step(np.zeros(self.num_envs, dtype=int))[-1]
+    if info["lives"].sum() > 0:
+      self.has_lives = True
+      print("env has lives")
 
   def reset(self, **kwargs):
     observations = super(RecordEpisodeStatistics, self).reset(**kwargs)
     self.episode_returns = np.zeros(self.num_envs, dtype=np.float32)
     self.episode_lengths = np.zeros(self.num_envs, dtype=np.int32)
+    self.lives = np.zeros(self.num_envs, dtype=np.int32)
     self.returned_episode_returns = np.zeros(self.num_envs, dtype=np.float32)
     self.returned_episode_lengths = np.zeros(self.num_envs, dtype=np.int32)
     return observations
 
   def step(self, action):
     observations, rewards, dones, infos = super(RecordEpisodeStatistics,
                                                 self).step(action)
-    self.episode_returns += rewards
+    self.episode_returns += infos["reward"]
```

Comment:
@vwxyzjn was this the reward bug you are alluding to? Since there was no description in the PR, I am not sure if this is the bug or if the additional code abotu `self.has_lives` is the bug?